### PR TITLE
Article on What's a SRV record.

### PR DESCRIPTION
**Pull request by aeden**

### CODE REVIEW

File: content/articles/srv-record.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: What is a SRV Record?
+excerpt: Welcome to DNSimple. This page is about the SRV record. Hosted DNS has never been this easy.
+categories:
+- DNS
+---
+
+# What is a SRV Record?
+
+An SRV record defines a Service.
```
Comment (sbastn) [inline content/articles/srv-record.markdown]: "Service" is kinda generic and can be confused with our one-click "Services". The paragraph below this one is much better. Either be more specific about what type of service, or consider removing this sentence.

**Comment by sbastn** [comment]
I have inlined one comment. Other than that, :clap: :rocket:

File: content/articles/srv-record.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: What is a SRV Record?
+excerpt: Welcome to DNSimple. This page is about the SRV record. Hosted DNS has never been this easy.
```
Comment (weppos) [inline content/articles/srv-record.markdown]: This excerpt... :scream:

Comment (aeden) [inline content/articles/srv-record.markdown]: :trollface:

**Pull request merged**